test/lish/uses_relative_resolution.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
```

Comment:
I think the file name has to end with `_test.dart` for the framework to pick it up.

Reply:
Ah, yes. I had been running `pub run test <filename>` directly, and didn’t notice. I’ll fix that.

Reply:
Also, it looks like there’s a failing test or two in the last commit that I didn’t see before, so I’ll have to check that out.